kobuki_dock_drive/src/dock_drive_states.cpp

```diff
@@ -76,9 +76,7 @@ namespace kobuki {
    *  @rotated       - indicates how much the robot has rotated while scan
    ********************************************************/
   void DockDrive::scan(RobotDockingState::State& nstate,double& nvx, double& nwz, const std::vector<unsigned char>& signal_filt, const ecl::LegacyPose2D<double>& pose_update, std::string& debug_str) {
-    unsigned char right = signal_filt[0];
```

Comment:
Be good to leave these in commented out for informative purposes.

Reply:
Restored in 3c23381